src/en/rizzcomic/src/eu/kanade/tachiyomi/extension/en/rizzcomic/RizzComic.kt

```diff
@@ -21,14 +21,16 @@ import rx.Observable
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class RizzComic : MangaThemesiaAlt(
-    "Rizz Comic",
-    "https://rizzfables.com",
+class RealMoasis : MangaThemesiaAlt(
+    "Real Moasis",
```

Comment:
Umm, I believe it's supposed to be "Realm Oasis" rather than "Real Moasis"...

Reply:
> Umm, I believe it's supposed to be "Realm Oasis" rather than "Real Moasis"...

haha, it did, I misread it and have now changed it!